### PR TITLE
Include `Site#recent_posts` in the current site API

### PR DESCRIPTION
**Pull request by tricknotes**
connects to https://github.com/bm-sms/daimon-news-misc/issues/109

### CODE REVIEW

File: app/models/site.rb [hunk 1]
```diff
@@ -27,6 +27,10 @@ def credit_enabled?
     participants.exists? && credit_roles.exists?
   end
 
+  def recent_posts
+    posts.published.order_by_recently.limit(5)
```
Comment (okkez) [inline app/models/site.rb:31]: Can we parameterize  `5`?
I want to specify limit in view.

File: app/serializers/site_serializer.rb [hunk 2]
```diff
@@ -22,4 +22,11 @@ class SiteSerializer < ActiveModel::Serializer
 
   has_many :categories
   has_many :links
+
+  has_many :recent_posts do
+    link :related do
+      api_site_posts_path(object.fqdn, page: {size: 5})
```
Comment (okkez) [inline app/serializers/site_serializer.rb:28]: Can we parameterize `size: 5` ?
(API client can specify how many recent posts in view.)

---

APIクライアントの側で表示する件数を決められるとよいと思います。

Comment (tricknotes) [inline app/serializers/site_serializer.rb:28]: This use case is a good point 👍 
I'll try it.

---

具体的なユースケースを添えてくれてありがとうございます！
外から件数を指定したいモチベーションがよくわかりました。

Comment (cafedomancer) [inline app/serializers/site_serializer.rb:28]: As an endpoint to fetch recent posts, using `/posts` looks like more natural (or more RESTful?) instead of exposing the endpoint `/recent_posts`,  such as `/posts` with query parameters (a sort field and a count), `/posts/recent` with query parameters (a count) and so forth. It just popped up in my mind.

Comment (cafedomancer) [inline app/serializers/site_serializer.rb:28]: Ah, `/recent_posts` doesn't exist. Sorry for my misunderstanding.

File: app/models/site.rb (hunk 1 above)
Comment (mtsmfm) [inline app/models/site.rb:31]: OK, let revert this change

File: app/serializers/site_serializer.rb (hunk 2 above)
Comment (mtsmfm) [inline app/serializers/site_serializer.rb:28]: I decided to not include this link.
`/posts` with `{page: {number: 5}}` is enough implementation at this time.

**Comment by mtsmfm** [comment]
I noticed this PR is not needed any more because order by recently is already implemented via https://github.com/bm-sms/daimon-news/pull/522
I'll close this PR
